pyanaconda/installation.py

```diff
@@ -90,34 +89,28 @@ def _prepare_configuration(storage, payload, ksdata):
     os_config = TaskQueue("Installed system configuration", N_("Configuring installed system"))
     os_config.append(Task("Configure authselect", ksdata.authselect.execute))
 
+    # add installation tasks for the Security DBus module
```

Comment:
Please remove these comments. By this comment you are telling the same as you can easily read from the code. If you want to have a comment here, then merge them together to ``add installation tasks from DBus modules``.

Reply:
The order of the modules is sometimes important & it simply seems more readable this way.

Reply:
Both ways are good to me. One is just slightly better.
Still you have my ACK.